fesvr/tsi.cc

```diff
@@ -27,16 +27,8 @@ tsi_t::~tsi_t(void)
 void tsi_t::reset()
 {
   uint32_t one = 1;
-  addr_t ipis[NHARTS_MAX];
-  int ncores = get_ipi_addrs(ipis);
 
-  if (ncores == 0) {
-      fprintf(stderr, "ERROR: No cores found\n");
-      abort();
-  }
-
-  for (int i = 0; i < ncores; i++)
-    write_chunk(ipis[i], sizeof(uint32_t), &one);
+  write_chunk(0x2000000, sizeof(uint32_t), &one);
```

Comment:
Yeah the hard-coded value feels bad to me as well. I think it should be a define. 
I'm not sure what the "right" solution here is. It seems acceptable that host software working with a tethered system is just supposed to know where everything is rather than inspecting the target. The reading of the config string previously made for a simple way to give all that information to the host software. At some point in the future it will probably seem convenient to be able to parse the new format (whatever it is).